Mathlib/Combinatorics/SimpleGraph/Maps.lean

```diff
@@ -275,8 +273,6 @@ instance instFintype [DecidableEq V] [Fintype V] [Fintype W] [DecidableRel G.Adj
     { toFun := fun f ↦ ⟨f.1, f.2⟩, invFun := fun f ↦ ⟨f.1, f.2⟩,
       left_inv := fun _ ↦ rfl, right_inv := fun _ ↦ rfl }
 
-instance [Finite V] [Finite W] : Finite (G →g H) := DFunLike.finite _
```

Comment:
Where is this gone?

Reply:
Done.

Reply:
(I removed it in the first commit because it's not needed anywhere in mathlib and because it alone makes `Maps` depend on `Data.FunLike.Fintype`.)